pandapower/file_io.py

```diff
@@ -407,3 +422,58 @@ def from_sqlite(filename, netname=""):
     net = from_sql(con)
     con.close()
     return net
+
+
+def from_postgresql(schema, host, user, password, database, include_results=False, **id_columns):
```

Comment:
PostgreSQL authomatically creates the schema public, which is used by default if the user does not specify the schema in a query.

I think it is better if the users have to think and make sure they want to use this default schema "public".

Let's see if other reviewers also see it that way, otherwise I can set the default value "public" for the parameter schema in the functions.